merge_audio_icd.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you not include this file in this PR?  It's not ready yet, and it's really something completely different.  That will keep the history cleaner.